orchestration/dagster_orchestration/hca_orchestration/resources/base.py

```diff
@@ -13,6 +13,15 @@
 POLLING_INTERVAL = 5  # seconds
 
 
+def default_google_access_token():
+    # get token for google-based auth use, assumes application default credentials work for specified environment
+    credentials, _ = google.auth.default()
+    auth_req = Request()
+    credentials.refresh(auth_req)
```

Comment:
can probably just pass `Request()` straight to `credentials.refresh` instead of the intermediate var since `auth_req` isn't used elsewhere, right?